monkey/monkey_island/cc/ui/src/components/ui-components/SearchBar.js

```diff
@@ -23,28 +26,22 @@ const SearchBar = (props) => {
   }
 
   return (
-    <Box>
+    <Box className="search-bar-wrapper">
       <TextField id={`search-bar-${nanoid()}`}
+                 fullWidth
                  variant={variant}
                  label={label}
                  value={currentValue}
                  placeholder={placeholder}
                  onChange={handleValueChange}
                  InputProps={{
-                   startAdornment: (
-                     <InputAdornment position="start">
+                   startAdornment: (<InputAdornment position="start">
```

Comment:
The previous formatting made more sense to me. Is this the result of running a formatter?